sway-lib-std/src/hash.sw

```diff
@@ -18,11 +18,19 @@ impl Hasher {
     }
 
     pub fn sha256(self) -> b256 {
-        self.bytes.sha256()
+        let mut result_buffer = b256::min();
```

Comment:
No docs?

Reply:
This can be done in another PR for the entire file.